backend/lambda/simplePing/index.js

```diff
@@ -0,0 +1,50 @@
+//TODO: Connect DynamoDB destination
```

Comment:
This was supposed to be at @crystalbroderick 's PR. I'm not sure why it's showing up here. Hopefully, there won't cause merge conflicts later to main branch:)